2020-05-03 23:24:15,987:DEBUG:certbot._internal.main:certbot version: 1.3.0
2020-05-03 23:24:15,987:DEBUG:certbot._internal.main:Arguments: ['--agree-tos', '--no-eff-email', '-m', 'przemek.pokrywka@gmail.com', '--server', 'https://acme-staging-v02.api.letsencrypt.org/directory', '--manual', '--manual-public-ip-logging-ok', '--preferred-challenges', 'dns', '--manual-auth-hook', '/bot/certbotauthenticator', '--manual-cleanup-hook', '/bot/certbotauthenticator', '-d', 'a.build.kyma-project.io.']
2020-05-03 23:24:15,987:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2020-05-03 23:24:16,343:DEBUG:certbot._internal.log:Root logging level set at 20
2020-05-03 23:24:16,344:INFO:certbot._internal.log:Saving debug log to /var/log/letsencrypt/letsencrypt.log
2020-05-03 23:24:16,352:DEBUG:certbot._internal.plugins.selection:Requested authenticator manual and installer None
2020-05-03 23:24:16,359:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * manual
Description: Manual configuration or run your own shell scripts
Interfaces: IAuthenticator, IPlugin
Entry point: manual = certbot._internal.plugins.manual:Authenticator
Initialized: <certbot._internal.plugins.manual.Authenticator object at 0x7feaf5dd40a0>
Prep: True
2020-05-03 23:24:16,360:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.manual.Authenticator object at 0x7feaf5dd40a0> and installer None
2020-05-03 23:24:16,361:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator manual, Installer None
2020-05-03 23:24:16,383:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-staging-v02.api.letsencrypt.org/acme/acct/13474575', new_authzr_uri=None, terms_of_service=None), 0ca10773bedaf0252bc041e7c5277c08, Meta(creation_dt=datetime.datetime(2020, 5, 3, 22, 46, 17, tzinfo=<UTC>), creation_host='1fe450f9cd72'))>
2020-05-03 23:24:16,385:DEBUG:acme.client:Sending GET request to https://acme-staging-v02.api.letsencrypt.org/directory.
2020-05-03 23:24:16,387:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-staging-v02.api.letsencrypt.org:443
2020-05-03 23:24:17,139:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 724
2020-05-03 23:24:17,140:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 03 May 2020 23:24:17 GMT
Content-Type: application/json
Content-Length: 724
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "keyChange": "https://acme-staging-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.2-November-15-2017.pdf",
    "website": "https://letsencrypt.org/docs/staging-environment/"
  },
  "nc2Yb-UVmws": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "newAccount": "https://acme-staging-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-staging-v02.api.letsencrypt.org/acme/new-order",
  "revokeCert": "https://acme-staging-v02.api.letsencrypt.org/acme/revoke-cert"
}
2020-05-03 23:24:17,146:INFO:certbot._internal.main:Obtaining a new certificate
2020-05-03 23:24:17,168:DEBUG:certbot.crypto_util:Generating key (2048 bits): /etc/letsencrypt/keys/0006_key-certbot.pem
2020-05-03 23:24:17,177:DEBUG:certbot.crypto_util:Creating CSR: /etc/letsencrypt/csr/0006_csr-certbot.pem
2020-05-03 23:24:17,179:DEBUG:acme.client:Requesting fresh nonce
2020-05-03 23:24:17,180:DEBUG:acme.client:Sending HEAD request to https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce.
2020-05-03 23:24:17,370:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2020-05-03 23:24:17,372:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 03 May 2020 23:24:17 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 0002ZdVI-USXMMduOUdEWMWUVOEBBwdoJOhLiKwNaldrBDQ
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2020-05-03 23:24:17,374:DEBUG:acme.client:Storing nonce: 0002ZdVI-USXMMduOUdEWMWUVOEBBwdoJOhLiKwNaldrBDQ
2020-05-03 23:24:17,376:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "a.build.kyma-project.io"\n    }\n  ]\n}'
2020-05-03 23:24:17,379:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xMzQ3NDU3NSIsICJub25jZSI6ICIwMDAyWmRWSS1VU1hNTWR1T1VkRVdNV1VWT0VCQndkb0pPaExpS3dOYWxkckJEUSIsICJ1cmwiOiAiaHR0cHM6Ly9hY21lLXN0YWdpbmctdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9uZXctb3JkZXIifQ",
  "signature": "TZwMY_O_tjFRroGB-1rKbJ_wr9I4lazd6wqHMS6I9rwTqxGkjjGVxKUASGBQSdX23TMVeiPM3OhBj3DdRtA3HscL77oNppp5Jr428W5M07ZCNXSQTg3bjMn571azPZy9tTCnGfiwrUAjAtSxMghK9E_Q7OaoLtCFrkTaV_PUsLdIzaWFhTh1towUqXjl26JVc5E1jX4mvr1xiBTIElVl2WoUbuyl2phHuibG1nwzglCJuH-hwz84heP7vvMmOcfGtkeR0bRi1hGdnjhBwOqnuu0GnLJSnAtGGFH7zDXBS8EaQRlRfsY1ZQkzJfpSR_1pYlWONJZ-5TQjNx5Zf0aM6w",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogImEuYnVpbGQua3ltYS1wcm9qZWN0LmlvIgogICAgfQogIF0KfQ"
}
2020-05-03 23:24:17,538:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 355
2020-05-03 23:24:17,541:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Sun, 03 May 2020 23:24:17 GMT
Content-Type: application/json
Content-Length: 355
Connection: keep-alive
Boulder-Requester: 13474575
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/order/13474575/89380797
Replay-Nonce: 0001tiSi-WxHxFEbA1Y7kfRbTj6dk4aQFna8KNumeUGaqbo
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2020-05-10T22:46:17Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "a.build.kyma-project.io"
    }
  ],
  "authorizations": [
    "https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/53674078"
  ],
  "finalize": "https://acme-staging-v02.api.letsencrypt.org/acme/finalize/13474575/89380797"
}
2020-05-03 23:24:17,543:DEBUG:acme.client:Storing nonce: 0001tiSi-WxHxFEbA1Y7kfRbTj6dk4aQFna8KNumeUGaqbo
2020-05-03 23:24:17,545:DEBUG:acme.client:JWS payload:
b''
2020-05-03 23:24:17,547:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/53674078:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xMzQ3NDU3NSIsICJub25jZSI6ICIwMDAxdGlTaS1XeEh4RkViQTFZN2tmUmJUajZkazRhUUZuYThLTnVtZVVHYXFibyIsICJ1cmwiOiAiaHR0cHM6Ly9hY21lLXN0YWdpbmctdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9hdXRoei12My81MzY3NDA3OCJ9",
  "signature": "ODXoV6qSQO3sQlDgT1ICWN7gvsW7HVLAxXpyzDDSrPrPxDB5MoSGSGoaqzzwvEtFwuN19FT8INCVrMApnGb_WPqUJrZueGmDUKaiI06kO_3ipXA6khK473gQ0w_EZMxw4pXikXJXQ8tLgkEihbqPn6dwrX9BukxrJr3YsLVftz3ghGYKbZYVWSrZvxJJR1OjkS7PFkqvxuWBgckBEV-3scYOwbRreLNLC-AMFyJ71iQd73JqDmhaGl9NlXKBX4i5gAJJbGgIPzHpzfykmEImT-FG6aVjmOsrl-7sbk6VFAM93gF3PAySZEHDslio_IbYXC-HX5zrEGR3jNdFScoxQg",
  "payload": ""
}
2020-05-03 23:24:17,738:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/53674078 HTTP/1.1" 200 819
2020-05-03 23:24:17,740:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 03 May 2020 23:24:17 GMT
Content-Type: application/json
Content-Length: 819
Connection: keep-alive
Boulder-Requester: 13474575
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 00029s7rHoisI-UOH-mE19Zvfzs80LqKHeZD5CujndQYEPI
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "a.build.kyma-project.io"
  },
  "status": "pending",
  "expires": "2020-05-10T22:46:17Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/53674078/k-IiAw",
      "token": "O7KrprNdoYPEngG60zf3vIJbmgRlVgmHvLzJGgaU_GQ"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/53674078/czGg-w",
      "token": "O7KrprNdoYPEngG60zf3vIJbmgRlVgmHvLzJGgaU_GQ"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/53674078/ugxT7g",
      "token": "O7KrprNdoYPEngG60zf3vIJbmgRlVgmHvLzJGgaU_GQ"
    }
  ]
}
2020-05-03 23:24:17,741:DEBUG:acme.client:Storing nonce: 00029s7rHoisI-UOH-mE19Zvfzs80LqKHeZD5CujndQYEPI
2020-05-03 23:24:17,743:INFO:certbot._internal.auth_handler:Performing the following challenges:
2020-05-03 23:24:17,745:INFO:certbot._internal.auth_handler:dns-01 challenge for a.build.kyma-project.io
2020-05-03 23:24:17,748:INFO:certbot._internal.hooks:Running manual-auth-hook command: /bot/certbotauthenticator
2020-05-03 23:24:17,757:ERROR:certbot._internal.hooks:manual-auth-hook command "/bot/certbotauthenticator" returned error code 127
2020-05-03 23:24:17,758:ERROR:certbot._internal.hooks:Error output from manual-auth-hook command certbotauthenticator:
/bin/sh: /bot/certbotauthenticator: not found

2020-05-03 23:24:17,761:INFO:certbot._internal.auth_handler:Waiting for verification...
2020-05-03 23:24:17,762:DEBUG:acme.client:JWS payload:
b'{\n  "resource": "challenge",\n  "type": "dns-01"\n}'
2020-05-03 23:24:17,766:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/53674078/czGg-w:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xMzQ3NDU3NSIsICJub25jZSI6ICIwMDAyOXM3ckhvaXNJLVVPSC1tRTE5WnZmenM4MExxS0hlWkQ1Q3VqbmRRWUVQSSIsICJ1cmwiOiAiaHR0cHM6Ly9hY21lLXN0YWdpbmctdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9jaGFsbC12My81MzY3NDA3OC9jekdnLXcifQ",
  "signature": "Vr7oR6jGcweoJGt8QY5hbnPJxz2Fjf4Dq3Mn4kmf4Aws96W3ddBXE-rCnhuTbIxcpNThoGBvnhS2wrV_wt56n5Mw4Iuezpi7HWr3pvifjiDhG_2P2S7BECiEA20AyoQs5Kf0gvbs4qam10vAlSJvML28BQKbAYcBDSuvgvjGFOep4MLA7rhtCH-RWMBUP332isWIvXR2nKCer4gEFh5szYobFw7d7BhoNCrApj336fHTx_7jKRzsaQCv6uV0Zn3CVdZ41UAXnnnK51PBsF7xThKX31LcgRdRAmrDZ-d7HVWaH16kGUsnLDCaV6UWT8mwbVEC-WOL9Du0AGkbGGb3Aw",
  "payload": "ewogICJyZXNvdXJjZSI6ICJjaGFsbGVuZ2UiLAogICJ0eXBlIjogImRucy0wMSIKfQ"
}
2020-05-03 23:24:17,962:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/chall-v3/53674078/czGg-w HTTP/1.1" 200 190
2020-05-03 23:24:17,965:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 03 May 2020 23:24:17 GMT
Content-Type: application/json
Content-Length: 190
Connection: keep-alive
Boulder-Requester: 13474575
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/53674078>;rel="up"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/53674078/czGg-w
Replay-Nonce: 0001Bx47pSDQzKJviACFCIJCF8Vk5Gx-6kKSVMBMCexXaBI
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "dns-01",
  "status": "pending",
  "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/53674078/czGg-w",
  "token": "O7KrprNdoYPEngG60zf3vIJbmgRlVgmHvLzJGgaU_GQ"
}
2020-05-03 23:24:17,966:DEBUG:acme.client:Storing nonce: 0001Bx47pSDQzKJviACFCIJCF8Vk5Gx-6kKSVMBMCexXaBI
2020-05-03 23:24:18,974:DEBUG:acme.client:JWS payload:
b''
2020-05-03 23:24:18,978:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/53674078:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xMzQ3NDU3NSIsICJub25jZSI6ICIwMDAxQng0N3BTRFF6S0p2aUFDRkNJSkNGOFZrNUd4LTZrS1NWTUJNQ2V4WGFCSSIsICJ1cmwiOiAiaHR0cHM6Ly9hY21lLXN0YWdpbmctdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9hdXRoei12My81MzY3NDA3OCJ9",
  "signature": "Dyj-ItsOiU5bYXEusV4zIN3LC_kz0iVCKkjOpo_qtO7bf7VbSYtPcVwSbp36WmxXqmZuQD2Z66DqtHcPW0h64OiuJHRSXM9xDHt2YsP9dz86AywhlDMsHwrKhqtier9ONccjOZYImfhnsCoCXncSDtTrkMeFZS_W2vfV0HoItCKJGn8fBUPBh3sIf0qq7w1FO0pIrS1PmY4QfxCZxbc6_2TJ_CnvU-N4HKFfIu7td10OQMmRgfjeSpuwaSd_cQrBE6xvDKwQhPPcvQfG9nB5TgmCKFUw7MKMmxvWyUcN2n38F3avOz8ivBy5WeFGPD1cHJnNCEJijQjC4f44l5jZRQ",
  "payload": ""
}
2020-05-03 23:24:19,169:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/53674078 HTTP/1.1" 200 630
2020-05-03 23:24:19,172:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 03 May 2020 23:24:19 GMT
Content-Type: application/json
Content-Length: 630
Connection: keep-alive
Boulder-Requester: 13474575
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 0001Iki0x5fBBUP1sSyePzqsykIl1pBZ-NpJ2-Xx6UE29s0
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "a.build.kyma-project.io"
  },
  "status": "invalid",
  "expires": "2020-05-10T22:46:17Z",
  "challenges": [
    {
      "type": "dns-01",
      "status": "invalid",
      "error": {
        "type": "urn:ietf:params:acme:error:dns",
        "detail": "DNS problem: NXDOMAIN looking up TXT for _acme-challenge.a.build.kyma-project.io - check that a DNS record exists for this domain",
        "status": 400
      },
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/53674078/czGg-w",
      "token": "O7KrprNdoYPEngG60zf3vIJbmgRlVgmHvLzJGgaU_GQ"
    }
  ]
}
2020-05-03 23:24:19,173:DEBUG:acme.client:Storing nonce: 0001Iki0x5fBBUP1sSyePzqsykIl1pBZ-NpJ2-Xx6UE29s0
2020-05-03 23:24:19,175:WARNING:certbot._internal.auth_handler:Challenge failed for domain a.build.kyma-project.io
2020-05-03 23:24:19,177:INFO:certbot._internal.auth_handler:dns-01 challenge for a.build.kyma-project.io
2020-05-03 23:24:19,182:DEBUG:certbot._internal.reporter:Reporting to user: The following errors were reported by the server:

Domain: a.build.kyma-project.io
Type:   dns
Detail: DNS problem: NXDOMAIN looking up TXT for _acme-challenge.a.build.kyma-project.io - check that a DNS record exists for this domain
2020-05-03 23:24:19,185:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 91, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 180, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2020-05-03 23:24:19,186:DEBUG:certbot._internal.error_handler:Calling registered functions
2020-05-03 23:24:19,188:INFO:certbot._internal.auth_handler:Cleaning up challenges
2020-05-03 23:24:19,190:INFO:certbot._internal.hooks:Running manual-cleanup-hook command: /bot/certbotauthenticator
2020-05-03 23:24:19,198:ERROR:certbot._internal.hooks:manual-cleanup-hook command "/bot/certbotauthenticator" returned error code 127
2020-05-03 23:24:19,200:ERROR:certbot._internal.hooks:Error output from manual-cleanup-hook command certbotauthenticator:
/bin/sh: /bot/certbotauthenticator: not found

2020-05-03 23:24:19,202:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 11, in <module>
    load_entry_point('certbot', 'console_scripts', 'certbot')()
  File "/opt/certbot/src/certbot/certbot/main.py", line 15, in main
    return internal_main.main(cli_args)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1347, in main
    return config.func(config, plugins)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1233, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 121, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 410, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 344, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 391, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 91, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 180, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
